src/routes/logout/+page.svelte

```diff
@@ -0,0 +1,26 @@
+<script lang="ts">
+  import { enhance } from "$app/forms";
+  import { getCurrentUserStore } from "$lib/context/currentUser";
+
+  export let form;
+
+  const currentUser = getCurrentUserStore();
+
+  $: if (form?.success) currentUser.set(undefined);
+</script>
+
+<div class="grid-container">
+  <h1>Logout</h1>
+  {#if $currentUser}
+    {#if form?.success}
+      <p>Successfully logged out!</p>
+    {:else}
+      <p>You are logged in as {$currentUser.name}.</p>
```

Comment:
this is related to my question in the complementary `+page.server.ts` file for this route